libs/cli/src/utils/config.ts

```diff
@@ -0,0 +1,30 @@
+import { type Tree, readJson } from '@nx/devkit';
+import { prompt } from 'enquirer';
+
+const configPath = 'components.json';
+
+export type Config = {
+	componentsPath: string;
+};
+
+export async function getOrCreateConfig(tree: Tree) {
+	if (tree.exists(configPath)) {
+		return readJson(tree, configPath) as Promise<Config>; // TODO: Parse with zod and handle errors
+	}
+
+	console.log('Configuration file not found, creating a new one...');
+
+	const { componentsPath } = (await prompt({
+		type: 'input',
+		required: true,
+		name: 'componentsPath',
+		message: 'Choose a directory to place your spartan libraries, e.g. libs/ui',
+		initial: 'libs/ui',
```

Comment:
I like the initial value, because some people assume that libs/ui is the default if they don't type it out!